capabilities/loki/chart/values.yaml

```diff
@@ -1,2 +1,2 @@
 grafana:
-  enabled: false
+  enabled: true
```

Comment:
nit: we could probably remove this value (along with the conditional wrapper around https://github.com/defenseunicorns/uds-core/blob/add_monitoring_and_logging/capabilities/loki/chart/templates/loki-dashboards.yaml). I don't think it provides a ton of value since the configmap won't break anything if deployed without grafana, and we don't want there to be any cases where grafana isn't deployed in our bundle.